src/groovy/org/transmartproject/db/dataquery/highdim/acgh/AcghModule.groovy

```diff
@@ -40,6 +40,7 @@ import org.transmartproject.db.dataquery.highdim.correlations.SearchKeywordDataC
 import org.transmartproject.db.dataquery.highdim.parameterproducers.AllDataProjectionFactory
 import org.transmartproject.db.dataquery.highdim.parameterproducers.DataRetrievalParameterFactory
 import org.transmartproject.db.dataquery.highdim.parameterproducers.MapBasedParameterFactory
+import static org.transmartproject.db.util.GormWorkarounds.createCriteriaBuilder
```

Comment:
I know it looks confusing that there is no any other change in the file. But actually this import is used. See firs line of `prepareDataQuery ` method body.